spec/pubid_jis/identifier_spec.rb

```diff
@@ -65,6 +65,20 @@ module Pubid::Jis
       it_behaves_like "converts pubid to pubid"
     end
 
+    context "JISX0902-1:2019" do
```

Comment:
Please add a comment here that this test is supposed to test an identifier without spaces.